index.html

```diff
@@ -174,73 +174,74 @@ <h3 class="sr-only">Math editor</h3>
+                                                data-step="3"
+                                                data-intro="Update the step.">
+                                                <span class="fa fa-pencil"></span>
+                                                Update Step
+                                            </button>
+                                            <br/>
+                                            <h3 class="sr-only">Cancel edit</h3>
+                                            <button id='cancelEdit' class="btn btn-default pointer editorButton" data-toggle="tooltip"
+                                                title="Cancel edit" onclick="ExitUpdate();"
+                                                data-step="3"
+                                                data-intro="Cancel edit.">
+                                                <span class="fa fa-ban"></span>
+                                                Cancel edit
+                                            </button>
+                                        </div>
+                                    </div>
+                                </div>
+
+                                <div class="problemFooter">
+                                    <div></div>
+                                    <div class="row">
+                                        <div id="control-buttons" class="col-lg-12 text-right">
+                                            <button class="btn btn-default pointer" id="BtnDiscard" onclick="GoogleAnalytics('Discard');">
+                                                Discard
+                                            </button>
+                                            <button class="btn btn-default pointer" id="BtnSave" data-step="5"
+                                                    data-intro="Save your work or close out to try again from the beginning."
+                                                    onclick="GoogleAnalytics('Save');">
+                                                    <span class="fa fa-thumbs-up"></span>
+                                                    Done
+                                            </button>
+                                            <button class="btn btn-default pointer" id="BtnClearAll" onclick="clearAllSteps()">
+                                                <span class="fa fa-times-circle"></span>
+                                                Clear all
+                                            </button>
+                                        </div>
                                     </div>
+                                    <!-- 		    <button class="btn btn-default pointer float-right" id="BtnSave" data-step="5" data-intro="Save your work or close out to try again from the beginning." onclick="GoogleAnalytics('Save');">Save Problem</button>		  	 -->
```

Comment:
shouldn't we remove it?

Reply:
I am not sure. I have found it and left like it is. I will remove it